cmd/evm/internal/t8ntool/flags.go

```diff
@@ -30,10 +30,13 @@ var (
 		Name:  "trace",
 		Usage: "Output full trace logs to files <txhash>.jsonl",
```

Comment:
Looks like this flag is not effective as we just define file name with different rule `fmt.Sprintf("trace-%d-%v.json", txIndex, txHash.String()))`